modules/encounter.py

```diff
@@ -261,6 +284,12 @@ def handle_encounter(
         else:
             context.set_manual_mode()
             encounter_info.battle_action = BattleAction.CustomAction
+
+    elif EncounterValue.avoid == True:
```

Comment:
```suggestion
    elif encounter_info.value == EncounterValue.AvoidedPokemon:
```

I think you just need that so that you can get rid of all the `True/False`